core/ir/aarch64/codec.py

```diff
@@ -365,7 +365,7 @@ def reorder_key(t):
     for opcode in sorted(case):
         c.append('    case OP_%s:' % opcode)
         if opc_props[opcode].feat != 'BASE':
-            c.append('#       if !defined(BUILD_TESTS)')
+            c.append('#       if !defined(DR_HOST_NOT_TARGET) && !defined(STANDALONE_DECODER)')
```

Comment:
Let's again be careful about limiting things to STANDALONE_DECODER since the full DR library is sometimes (all the time, for our use case) used for decoding as it has to be the same library used for managed execution.

Reply:
I thought this issue has been resolved here:
https://github.com/DynamoRIO/dynamorio/pull/5835#discussion_r1087301519
https://github.com/DynamoRIO/dynamorio/pull/5835#discussion_r1087303056